udata/utils.py

```diff
@@ -139,6 +139,17 @@ def daterange_end(value):
         return result.replace(month=12, day=31)
 
 
+def date_or_aware_datetime_to_naive_datetime(given_date):
```

Comment:
To keep the `to_` pattern, we could maybename it like
```suggestion
def to_naive_datetime(given_date):
```